Gemfile.lock

```diff
@@ -448,4 +506,4 @@ DEPENDENCIES
   wirble
 
 BUNDLED WITH
-   1.13.6
+   1.15.1
```

Comment:
@mssola I didn't run `bundle update`, just updated bundle because of warning, it possible to return v 1.13.6